### DISCUSSION TITLE
React with JointJs

### DISCUSSION THREAD

**Discussion by Rafael-Balocco** [clientIO/joint#2622] [Q&A] [answered]
### Introduction

I've been searching for a way to integrate my completed React.ts app with Joint.js. 
Thus far, I've only come across examples demonstrating how to connect Joint.js + with React, but I haven't found a tutorial detailing how to install the open-source version of Joint.js. 
Does anyone know how to accomplish this?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
It's almost the same as *JointJS+* [tutorial](https://resources.jointjs.com/tutorial/react-ts) with the following differences:
- *JointJS* does not have CSS to include
- There is no `PaperScroller` in *JointJS*, so use the `Paper` only.

**Comment by Rafael-Balocco** [comment]
Thanks for the answer!
Sorry if the questions looks stupid, I am new at it.
What about the joint-plus.tgz and its dependency? do we have access to a similar file, like a joint.tgz using open source joint? How we deal with it?

**Reply by kumilingus** [reply]
Install the *JointJS* open-source package (`@joint/core`) using your package manager:

```sh
npm add @joint/core
```

And import features of JointJS into your application as necessary:

```js
import { dia, util, shapes } from '@joint/core';
```

**Reply by Rafael-Balocco** [reply]
Thank you for all the support! Everything is working now!
